Automattic/newspack-popups
# feat(inline prompt): add a new placement option for the archive pages

### All Submissions:






### Changes proposed in this Pull Request:



enable inserting prompts in after x articles or every x article in the archive pages

fix #450

### How to test the changes in this Pull Request:

1. If it's not released, please checkout https://github.com/Automattic/newspack-theme/pull/1507 or add that small change manually inside the `newspack-theme` that you use.
2. Pull this PR, and add a new `inline` prompt from the campaign menu.
3. In the `placement` option select `In archive pages`.
4. Set `2` for example as the number of items to show before the prompt.
5. Load an archive page (as a non-admin) and observe that the prompt is displayed after the second article in the list.
6. Edit that prompt, and switch on the `repeat prompt`.
7. Reload the archive page and observe that the prompt now is displayed after every two articles.

### Other information:
